spring-session/src/main/java/org/springframework/session/jdbc/JdbcOperationsSessionRepository.java

```diff
@@ -355,6 +360,8 @@ private static GenericConversionService createDefaultConversionService() {
 
 		private final ExpiringSession delegate;
 
+		private long lastAccessedTime;
```

Comment:
Is there a reason that we cannot just set and get the lastAccessedTime on the delegate?